### PR TITLE
Flame - loading multilayer with controlled layer names

### PR DESCRIPTION
**Pull request by jakubjezek001**
## Brief description
Layers can be renamed dynamically during loading to batch

## Description
Folowing settings were created for layer template and layer rename patterns:
- `project_settings/flame/load/LoadClipBatch/layer_rename_template`
- `project_settings/flame/load/LoadClipBatch/layer_rename_patterns`
image

Regular expression patterns could be used for detecting name patterns in loaded layers and if any match is found, then the layer is renamed to use of version context data and additional formatting keys found if the context of loading file and its layer. For example layer with name `rgba` and layer uid `BEAUTY:MasterBeauty` can be now renamed with template `{asset}_{subset} [{layerUID}]` to new name `shot01_renderMain [BEAUTY:MasterBeauty]`. 
image

This had been requested by a client who is using Flame for metadata slates. The metadata key layer will now show more than just `rgba`.

### CODE REVIEW

File: openpype/hosts/flame/plugins/load/load_clip.py [hunk 1]
```diff
@@ -38,8 +47,14 @@ def load(self, context, name, namespace, options):
         version_name = version.get("name", None)
         colorspace = self.get_colorspace(context)
 
+        # in case output is not in context replace key to representation
+        if not context["representation"]["context"].get("output"):
+            self.clip_name_template.replace("output", "representation")
```
Comment (iLLiCiTiT) [inline openpype/hosts/flame/plugins/load/load_clip.py:52]: There two lines won't do anything.  Replace called on string won't change the source string but will return new.

Comment (jakubjezek001) [reply openpype/hosts/flame/plugins/load/load_clip.py]: ah, yeah thanks

**Comment by jakubjezek001** [comment]
Ready for merge ;)

File: openpype/hosts/flame/plugins/load/load_clip.py [hunk 2]
```diff
@@ -38,8 +47,16 @@ def load(self, context, name, namespace, options):
         version_name = version.get("name", None)
         colorspace = self.get_colorspace(context)
 
+        # in case output is not in context replace key to representation
+        if not context["representation"]["context"].get("output"):
+            self.clip_name_template = self.clip_name_template.replace(
```
Comment (iLLiCiTiT) [inline openpype/hosts/flame/plugins/load/load_clip.py:52]: I just realized that if the load plugin is used for loading of 2+ representations and only one of them does not have `"output"` the template for others will be broken. My recommendation is to not change value on `self`.

**Review by iLLiCiTiT — APPROVED** [review]
Can't test if without flame but code looks ok.

**Pull request merged**